spring/build.gradle

```diff
@@ -0,0 +1,27 @@
+dependencies {
+    // TODO(ide) Remove thrift dependencies
+    compile project(':thrift')
```

Comment:
Probably OK to keep this as long as it's optional? If we remove this we can't support Thrift service registration.